src/pages/es/guides/fonts.md

```diff
@@ -103,11 +103,39 @@ El proyecto [Fontsource](https://fontsource.org/) simplifica el uso de Google Fo
     }
     ```
 
-## Más recursos
+## Agregando fuentes con Tailwind
+
+Si estás usando la [integración de Tailwind](/es/guides/integrations-guide/tailwind/), puedes añadir una declaración de `@font-face` para utilizar una fuente local o puedes usar la estrategia `import` de Fontsource para registrar tu fuente.
+
+1. Sigue los pasos de las guías anteriores, pero deja sin completar el paso final sobre agregar una `font-family` en tu CSS.
 
-### Añade fuentes usando Tailwind
+1.  Agrega el nombre de la tipografía en `tailwind.config.cjs`.
```

Comment:
```suggestion
2. Agrega el nombre de la tipografía en `tailwind.config.cjs`.
```

Reply:
Aquí creo que se tiene que cambiar el numeral porque estás listando pasos. ¿Qué opinas?

Reply:
Ah, I think this is copied from the English version. Markdown has a feature where a list numbered `1. ... 1. ... 1. ...` gets the correct numbers when rendered. Handy because if you add a new item in the middle of the list you don’t need to go through renumbering the subsequent items.

We can see this in action [in the deploy preview](https://deploy-preview-2218--astro-docs-2.netlify.app/es/guides/fonts/#agregando-fuentes-con-tailwind):

![image](https://user-images.githubusercontent.com/357379/208961463-8fb38861-b74f-4ba1-b1de-374a7d4fd1f5.png)